lib/3scale/api/client.rb

```diff
@@ -157,7 +157,7 @@ def proxy_config_latest(service_id, environment = 'sandbox')
       # @return [Hash]
       # @param [Fixnum] service_id Service ID
       # @param [String] environment. Must be 'sandbox' or 'production'
-      # @param [Fixnum] proxy configuration version
+      # @param [Fixnum] version Version to promote
```

Comment:
it is `show_proxy_config` method. Nothing to do with promoting.

`version` is the version id to be shown